textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToGramQueryTranslatorTest.java

```diff
@@ -0,0 +1,29 @@
+package edu.uci.ics.textdb.dataflow.regexmatch;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author Shuying Lai
+ * @author Zuozhi Wang
+ */
+
+public class RegexToGramQueryTranslatorTest {
```

Comment:
This test is too "weak."  Can you make it better?  Also those other related classes need tests as well.